Gemfile.lock

```diff
@@ -77,4 +89,4 @@ DEPENDENCIES
   vcr
 
 BUNDLED WITH
-   2.3.9
+   2.6.3
```

Comment:
Bundled with the latest stable version but is not compatible with version under `3.1` so used a different version in the workflow known to be compatible with `3.0.1`
Ref: https://bundler.io/compatibility.html

Reply:
Since 3.0.x is EOL, I agree we can just support >= 3.1.x.